docs/integration_resources/software/nuke.md

```diff
@@ -13,10 +13,68 @@ from shift.tools import nshift
 nshift.show()
 ```
 
+## Shift installation in Nuke
+
+In order to have Shift installed inside Nuke and be able to open the Shift UI with a menu entry, is required to setup Shift in the user init and menu python files. It can be added in the user init and menu files from the nuke preference folder ([home directory]/.nuke) or a custom init and menu files inside your pipeline. If the directory were this files are stored is a path read by Nuke, they will be automatically executed on startup.
+
+### Init
+
+In the init.py file is required to configure the Shift basics to work in Nuke batch and interface modes. For that purpose is only required to add the Shift installation path to the environment variables. 
+
+```python
+import sys
+sys.path.append("<path_to_your_shift_folder>")
+sys.path.append("<path_to_your_shift_folder>/shift/thirdparty/python/Lib/site-packages")
+```
+
+>[!NOTE]
+> If the paths are added in the system or before opening Nuke, then this configuration step will not be required.
+
+
+### Menu
+
+In the menu.py file it is required to configure the Shift menu and toolbar entries in the Nuke UI.
+
+
+```python
+import os
+import nuke
+
+try:
+    import shift
+    shiftPath = os.path.dirname(shift.__file__)  # Get the directory location of Shift
+    # Build the path to the shift_icon for the toolbar
+    iconPath = os.path.join(shiftPath, "icons", "shift_icon_window.png")
+    shiftCode = "from shift.tools import nshift;nshift.show()"
+    # Create the toolbar entry
+    menubar = nuke.menu("Nodes")
+    shiftM = menubar.addMenu("Shift", icon=iconPath)
+    shiftM.addCommand("Shift UI", shiftCode, '')
+    # Create a Menu entry
+    menubar = nuke.menu("Nuke")
+    shiftM = menubar.addMenu("Shift")
+    shiftM.addCommand("Shift UI", shiftCode, '')
+except Exception as e:
+    pass  # Skip the Error raise to avoid Nuke failing at opening if setting up the menu does not work on startup.
+```
+
+This code placed in a menu.py file executed by Nuke will create a Shift toolbar entry
+
+
+![Shift Toolbar](../../images/nuke/shift_toolbar.png)
+
+and a Shift menu entry
+
+![Shift Menu](../../images/nuke/shift_menu.png)

```

Comment:
Please, check how to add the html code from this card and apply it to this image: https://github.com/Inbibo/shift_docs/issues/45

Ensure it renders properly on the webpage.

Reply:
I have a problem with the images. I think I have setup them like in the Issue. However they display ok in github but not in the webpage. Do you see something wrong @juliavisserinbibo ?

Reply:
I do see them
![image](https://github.com/user-attachments/assets/643f2dec-f70c-4e52-b538-22336dd00867)

Reply:
Ok, maybe When I tryied it was cached in the web or something. Thx!

Reply:
To make sure you are looking at the latest changes I suggest looking at the file that is displayed in "Files change" since it corresponds to the file status in the latest commit. 
Because the path is unique to the commit and the link will be different, there will not be cache problems.
![inspectweb](https://github.com/user-attachments/assets/5f943edd-8640-4da6-9ea9-1df3746291bc)

Reply:
well, I checked again and still does not display
![image](https://github.com/user-attachments/assets/f74ffcb7-49e3-4677-bf4e-49d03ba88d5f)
really weird

Reply:
@Inbibo/shift can someone else try it out and see what the result is?

Reply:
On my side it's working fine too:
![image](https://github.com/user-attachments/assets/567d8f60-d0da-475f-8081-a7f5576626c4)